### PR TITLE
Added hJoe v2 token

### PR DESCRIPTION
**Pull request by blenessy**
A bit confusing as they have both a hJoe v1 and v2 tokens with the
same symbol. Since v2 deprecated v1, I will not bother adding it.

### CODE REVIEW

File: joe.tokenlist.json [hunk 1]
```diff
@@ -663,7 +663,15 @@
...
+    {
+      "chainId": 43114,
+      "address": "0xe7250b05bd8DEE615ecC681eda1196aDD5156F2B",
+      "decimals": 18,
+      "name": "hJoe v2",
+      "symbol": "hJoe",
+      "logoURI": "https://raw.githubusercontent.com/traderjoe-xyz/joe-tokenlists/main/logos/0xe7250b05bd8DEE615ecC681eda1196aDD5156F2B/logo.png"
+    }    
   ],
   "timestamp": "2021-08-06T00:00:00+00:00"
```
Comment (0xmurloc) [inline joe.tokenlist.json:676]: pls bump the date

**Review by 0xmurloc — CHANGES_REQUESTED** [review]

File: joe.tokenlist.json (hunk 1 above)
Comment (blenessy) [reply joe.tokenlist.json]: sure, np

Comment (blenessy) [reply joe.tokenlist.json]: done

**Comment by 0xmurloc** [comment]
added separate commit

**Pull request closed without merging**